src/lighteval/__main__.py

```diff
@@ -22,81 +20,36 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
+import typer
```

Comment:
new requirement?

Reply:
yes, I gonna add the requirements at the end